pkg/cloud/vsphere/services/govmomi/create.go

```diff
@@ -638,3 +646,12 @@ func byteToGiB(n int64) int64 {
 func giBToByte(n int64) int64 {
 	return int64(n * int64(math.Pow(1024, 3)))
 }
+
+// newMACAddr generates a new MAC address using VMware's OUI.
+func newMACAddr() (string, error) {
```

Comment:
Maybe for another PR, but we can unit test this by running it through `net.ParseMac`

Reply:
Unit test that we correctly formatted six bytes as a colon-separated hexadecimal string?

Reply:
No harm in that IMO, a unit test using `ParseMac` should be super simple. If you leave it out here let me know and happy to add it as a follow-up :)

Reply:
Fair enough. I'll add it in the commit used to address this round of feedback.

Reply:
This commit was removed from this PR